.travis.yml

```diff
@@ -9,6 +9,7 @@ cache:
   directories:
     - travis-build
 before_install:
+  - npm i npm@6.14.8
```

Comment:
you can remove this

Reply:
Just added that while resolving travis build issue. It is removed now. https://github.com/cerner/terra-graphs/pull/5/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R13